extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -119,6 +119,12 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
     return null;
   }
 
+  /** @override */
+  getSigningServiceNames_() {
+    // TODO(levitzky) Add dev key name once it goes live.
+    return getMode().localDev ? ['google'] : ['google'];
```

Comment:
I do think there is some value to have the URL defined in one place, and refer to it using a "name" everywhere else. I suppose it would save us some bytes in the binary.

I would imagine "google" signing server would be the most common one, if there will be anyone else.